README.md

```diff
@@ -124,32 +131,28 @@ ACCESS_KEY_ID=xxxxxxxxxxxx
 ACCESS_KEY_SECRET=xxxxxxxxxx
 ```
 
-建议将 .env 放到 .gitignore 中，避免泄漏重要的账户信息。
+It is recommended that put the `.env` file into `.gitignore` to protect your acouunt credentials from leaking into the public network environment.
```

Comment:
acouunt -> account

Reply:
It is recommended that add the `.env` into `.gitignore` file to prevent your account credentials be checked into code repository.